extensions/arc/package.nls.json

```diff
@@ -148,6 +148,7 @@
 	"arc.agreement.sql.terms.conditions": "Azure SQL managed instance - Azure Arc terms and conditions",
 	"arc.agreement.postgres.terms.conditions": "Azure Arc-enabled PostgreSQL Hyperscale terms and conditions",
 	"should.be.integer": "Value must be an integer",
+	"worker.node.count.should.not.be.one": "Value of 1 is not recommended",
```

Comment:
"not recommend" implies that they could do it anyways. If this is a valid value but we're just not allowing them to do it through the UI then I'd suggest making that clearer

Reply:
I am not sure if this is something that can still occur through CLI, waiting on confirmation for best validation response.